sw/device/tests/BUILD

```diff
@@ -1108,6 +1110,65 @@ opentitan_test(
     ],
 )
 
+# hash table for lc_state name and value.
+# 'name' is picked for pre generated otp image
+# 'value' is copied from
+#   https://cs.opensource.google/opentitan/opentitan/
+#   +/master:rules/const.bzl;drc=30f44025dfb4bc2bd328a99a0a810c3189fb2544;l=29
+_FLASH_CTRL_INFO_ACCESS_LC_STATES = {
+    "test_unlocked0": 0x02108421,
+    "dev": 0x21084210,
+    "prod": 0x2318c631,
+    "prod_end": 0x25294a52
+}
+
+load(
+    "//rules:opentitan.bzl",
+    "RSA_ONLY_KEY_STRUCTS",
+)
+load(
+    "//rules/opentitan:defs.bzl",
+    "rsa_key_for_lc_state",
+)
```

Comment:
The following should appear at the beginning of the file. Try running the following command to auto-format:

```console
bazel run //quality:buildifier_fix
```

Reply:
Moved to the beginning.